plutus-ledger-api/testlib/PlutusLedgerApi/Test/EvaluationEvent.hs

```diff
@@ -143,17 +144,15 @@ data TestFailure
 renderTestFailure :: TestFailure -> String
 renderTestFailure = \case
     InvalidResult err -> display err
-    MissingCostParametersFor ver ->
-        [fmt|
-Missing cost parameters for {show ver}. Report this as a bug
-against the script dumper in plutus-apps."
-|]
+    MissingCostParametersFor ver -> [__i|
+        Missing cost parameters for #{show ver}.
```

Comment:
i think calling show is redundant here, unless the extra double-quotes around the version are needed.